### PR TITLE
Improve process for Webapp local build

### LINKED ISSUE #62: Improve build process for local webapp
Currently, it is not that easy to build the webapp locally. In order to allow for easier testing, we should improve this process. Myself and @Anders-Markvardsen have already gone a long way to improving/creating instructions for [local setup of the webapp for development](https://github.com/ISISScientificComputing/autoreduce/wiki/Installation-for-local-development) but there are still some part that need to be completed:
* [x] Ensure the `test_settings.py` is up-to-date for the webapp
   * [x] update activeMq
   * [x] deal with `OPTIONS` for database (this should be added to the dev node settings if not already in use
   * [x] Add `127.0.0.1` to allowed host in settings
* [x] Create super user on local database
* ~Find and write script to acquire bootstrap static content~ *css is shipped with code base* 
* [x] ICAT install script for windows 
* [x] ActiveMQ install script for windows

### PR DESCRIPTION
**Pull request by ElliotAOram**
**Description of changes**

* Add batch file installations for activeMQ and icat
* Use local host settings for activemq in webapp test settings
* Ensure super user is created for local webapp login

**How to test**
* Follow the updated [webapp installation documentation](https://github.com/ISISScientificComputing/autoreduce/wiki/Installation-for-local-development) and check all the stages work on windows and linux

Fixes #62

### CODE REVIEW

**Comment by ElliotAOram** [comment]
@jsws how far did we get with this in the end? Are you waiting for me to change something

File: Scripts/Build/install/activemq.bat [hunk 1]
```diff
@@ -0,0 +1,22 @@
+setlocal EnableDelayedExpansion
+@echo off
+
+REM This script requires 7zip to be installed and in the system path.
+REM If you do not have 7zip you can download it from: https://www.7-zip.org/download.html
+
+REM Download and extract
+set target_path=%1
+set folder=%target_path%activemq
+set destination=%folder%\activmq.tar.gz
```
Comment (ElliotAOram) [inline Scripts/Build/install/activemq.bat]: this should be a `.zip` files not the `.tar.gz` linux version

File: Scripts/Build/install/icat.bat [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    echo "ICAT files already detected in this location"
+)
+
+REM install:
+cd %folder%\python-icat-0.13.1
+python setup.py build
+python setup.py install
+
+REM validate:
+python %~dp0\tests\validate_icat.py
```
Comment (jsws) [inline Scripts/Build/install/icat.bat]: This give the wrong location, the validate_icat.py is in `Scripts/Build/tests`. This gives `Scripts\Build\install\tests`
```
python: can't open file 'C:\Users\me\Downloads\autoreduce\Scripts\Build\install\tests\validate_icat.py': [Errno 2] No such file or directory
```

**Comment by ElliotAOram** [comment]
I remember now. The reason we use move over copy is to satisfy pylint:
* If you run pylint before the mv/cp - import error as settings.py doesn't exist
* If you run after cp pylint complains about duplicate code

Is there a more intelligent way to do this I wonder. I rather not have to ignore all the `import-error`s if we can avoid

**Comment by ElliotAOram** [comment]
Yes - we should just ignore settings files

**Comment by ElliotAOram** [comment]
This should be good to go now

**Comment by jsws** [comment]
Tested on Windows 7 and Ubuntu

**Pull request merged**